Example/Tests/ComponentAPICoordinatorTests.swift

```diff
@@ -9,17 +9,23 @@
 import XCTest
 @testable import Example_Swift
 @testable import GiniVision
+@testable import Gini_iOS_SDK
 
-class GINIComponentAPICoordinatorTests: XCTestCase {
+final class ComponentAPICoordinatorTests: XCTestCase {
     
     var componentAPICoordinator: ComponentAPICoordinator?
+
+    var documentService: ComponentAPIDocumentServiceProtocol = {
+        let sdkBuilder = GINISDKBuilder.anonymousUser(withClientID: "",
+                                                      clientSecret: "",
+                                                      userEmailDomain: "")
+       return DocumentServiceMock(sdk: sdkBuilder!.build()!)
```

Comment:
If it's a mock, why does it even require an SDK instance? It seems redundant and requires you to write some extra code above.

Reply:
Because both the initializer and the stored property are part of the protocol

Reply:
You don't have to follow the protocol since you are referring to the concrete subclass. You can use a convenience initializer to create and mock and then just use that it conforms to the document service protocol

Reply:
You are right, that could be an option.